core-tests/shared/src/test/scala/zio/FiberSpec.scala

```diff
@@ -162,7 +162,36 @@ object FiberSpec extends ZIOBaseSpec {
             currentFiber <- ZIO.succeed(Fiber.currentFiber())
           } yield assertTrue(currentFiber.isDefined)
         }
-      } @@ TestAspect.fromLayer(Runtime.enableCurrentFiber)
+      } @@ TestAspect.fromLayer(Runtime.enableCurrentFiber),
+      suite("FiberFailure stack trace handling")(
+        test("FiberFailure captures the full stack trace including user code") {
+          def subcall(): Unit =
+            Unsafe.unsafe { implicit unsafe =>
+              Runtime.default.unsafe.run(ZIO.fail("boom")).getOrThrowFiberFailure()
+            }
+          def call(): Unit = subcall()
+
+          val fiberFailureTest = ZIO
+            .attempt(call())
+            .catchAll {
+              case fiberFailure: FiberFailure =>
+                ZIO.succeed(fiberFailure.getStackTrace.mkString("\n"))
+              case other =>
+                ZIO.succeed(s"Unexpected failure: ${other.getMessage}")
+            }
+            .asInstanceOf[ZIO[Any, Nothing, String]]
+
+          fiberFailureTest.flatMap { stackTrace =>
+            ZIO.succeed {
+              assertTrue(
+                stackTrace.contains("call") &&
```

Comment:
This test is wrong. This will be true in cases that `call` is not in the trace but `subcall` is because `subcall` also has `call` in it. You need to rename `call` to `call1` or something

Reply:
```
zio.FiberSpec$.subcall$1(FiberSpec.scala:169)
zio.FiberSpec$.call$1(FiberSpec.scala:172)
```

Okay! Makes sense to avoid false positive results. Will modify it